darwin/dataset/remote_dataset.py

```diff
@@ -278,6 +278,13 @@ def pull(
                 annotations_dir.mkdir(parents=True, exist_ok=False)
                 stems: dict = {}
 
+                # If properties were exported, move the manifest file to the annotations folder
+                if (tmp_dir / ".v7").exists():
+                    metadata_file = tmp_dir / ".v7" / "metadata.json"
+                    metadata_dir = annotations_dir / ".v7"
+                    metadata_dir.mkdir(parents=True, exist_ok=False)
```

Comment:
- changing the `exist_ok=False` -> `True` - to make sure it dosen't give an error. Docs: https://docs.python.org/3/library/os.html#os.makedirs

```suggestion
                    metadata_dir.mkdir(parents=True, exist_ok=True)
```